www/templates/survey/enketo/timeuse-button.html

```diff
@@ -1,6 +1,5 @@
 <div class="fab-button fab-button-above-tab-bar">
-  <button
-    class="button icon ion-plus button-primary fab-button-shadow"
-    ng-click="openPopover($event)"
-  ></button>
+  <button class="button icon ion-plus button-primary fab-button-shadow" ng-click="openPopover($event)">
+    <b>Add Activity</b>
```

Comment:
This should be, at a minimum, internationalized. Please see the way that other text in the app is represented.
Ideally, we would also have this text come in as part of the config so that we can support other use cases like the "missing trip"

Reply:
I was wondering about that. will ask today